src/Files/App.xaml.cs

```diff
@@ -315,17 +320,30 @@ private Frame EnsureWindowIsInitialized()
             return rootFrame;
         }
 
-        private void CoreWindow_Activated(CoreWindow sender, WindowActivatedEventArgs args)
+        private async void CoreWindow_Activated(CoreWindow sender, WindowActivatedEventArgs args)
         {
             if (args.WindowActivationState == CoreWindowActivationState.CodeActivated ||
                 args.WindowActivationState == CoreWindowActivationState.PointerActivated)
             {
                 ShowErrorNotification = true;
-                ApplicationData.Current.LocalSettings.Values["INSTANCE_ACTIVE"] = Process.GetCurrentProcess().Id;
+
                 if (MainViewModel != null)
                 {
                     MainViewModel.Clipboard_ContentChanged(null, null);
                 }
+
+                var proc = Process.GetCurrentProcess();
+                ApplicationData.Current.LocalSettings.Values["INSTANCE_ACTIVE"] = proc.Id;
```

Comment:
TODO: docs suggest using `VisibilityChanged` instead of `Activated` to check if the prelaunch instance has been opened.

Reply:
Would that prevent the sidebar items from loading?

Reply:
It would (at least the icons and some of the sections), until the app gets actually open. I don't really know if leaving stuff like this would be a problem, it's just that the [docs](https://docs.microsoft.com/en-us/windows/uwp/launch-resume/handle-app-prelaunch) state: `Apps should not perform long running operations during prelaunch because the app will terminate if it can't be suspended quickly.`
But how long is "long"? Does it hold only for prelaunch by the system or also for our "manual" prelaunch?

Reply:
I have yet to test this in Release (Azure won't compile, @yaichenbaum do you know why?) but if it works I'd rather keep stuff as is (load everything).

Reply:
It takes a couple of seconds until it suspends the process in release mode.

Reply:
The pipeline is failing because it's trying to test the suspended process. @chingucoding is it possible to force the ci to test the main process and not the addiitonal process that's pre launched?

Reply:
Sorry for the late reply. While this is marked as resolved, are we sure that we succeeded the test on purpose or rather on accident? And if there is a way to find the main process, sure we can force testing the main process.

Reply:
100% solved by accident 😂 so perhaps there's a more robust way to handle this.

Reply:
The question would be how you find that window then. The TLDR of the test is that we need to find the process ID for the window so that Axe can scan the app correctly. Currently, we are just finding the first process with the name "Files".